rflx/model.py

```diff
@@ -280,13 +280,41 @@ class Enumeration(Scalar):
     def __init__(
         self,
         identifier: StrID,
-        literals: Dict[StrID, Number],
+        literals: Sequence[Tuple[StrID, Number]],
         size: Expr,
         always_valid: bool,
         location: Location = None,
     ) -> None:
         super().__init__(identifier, size, location)
 
+        for i1, e1 in enumerate(literals):
+            for i2, e2 in enumerate(literals):
+                if i2 < i1 and e1[0] == e2[0]:
+                    self.error.append(
+                        f'duplicate element "{e1[0]}"',
```

Comment:
I think changing it to "duplicate literal" would make the error message clearer.

Reply:
Done in 595f46b.